war/src/main/webapp/css/simple-page-forms.css

```diff
@@ -31,12 +35,17 @@
 }
 .Checkbox .svg-icon {
   opacity: 0;
-  fill: #fff;
```

Comment:
@oleg-nenashev @timja @fqueiruga 

Was there any reason to remove this "fill" ?

In the login page, the "Keep me signed in" checkbox is less constrasted now.

### Version 2.222

![Screenshot_2021-05-27_110854_001](https://user-images.githubusercontent.com/2662497/119798966-ef7d1780-bedb-11eb-9af4-369ba288bd8b.png)

### Version 2.277

![Screenshot_2021-05-27_110837_001](https://user-images.githubusercontent.com/2662497/119798943-e8560980-bedb-11eb-9f17-cb2c3bf32fe4.png)

Adjustment proposed in https://github.com/jenkinsci/jenkins/pull/5536

Reply:
no, accidental